packer/install-dependencies.sh

```diff
@@ -313,9 +313,9 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.0.31
 1.0.30
 1.0.29
-1.0.28
```

Comment:
Let's not remove 1.0.28 from the VHD as part of this change

Reply:
Is it ok to remove since 1.0.28 and below had quite a few bugs?